x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/SecurityRestFilter.java

```diff
@@ -88,6 +90,11 @@ public void handleRequest(RestRequest request, RestChannel channel, NodeClient c
                         e -> handleException("Secondary authentication", request, channel, e)));
                 }, e -> handleException("Authentication", request, channel, e)));
         } else {
+            if (request.method() != Method.OPTIONS) {
+                HeaderWarning.addWarning("Elasticsearch built-in security features are not enabled, your cluster may be accessible " +
+                    "without authentication. Read https://www.elastic.co/guide/en/elasticsearch/reference/" + Version.CURRENT.major + "." +
+                    Version.CURRENT.minor + "/get-started-enable-security.html for more information");
+            }
```

Comment:
I'll try to find out

Reply:
The [linked page](https://www.elastic.co/guide/en/elasticsearch/reference/current/get-started-enable-security.html) was removed as part of the recent security documentation overhaul. However, there is a redirect in place to [Set up minimal security for the Elastic Stack](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-minimal-setup.html), so I would use that link instead.

Hardcoded links can be brittle, but if the page is moved or changed, we implement redirects like the one for the `get-started-enable-security.html` page. The Kibana team uses a link service to manage doc links instead of hardcoding them, but that's a much heavier lift than adding a single link.